Can't install with pecl: src/ecma402/calendar.loT: No such file or directory

I'm trying to install this extension with pecl in a docker image, without success.

1. In a docker image launched with `docker run --rm -it php:8.2-cli-alpine3.17 sh`
   ```
   apk add $PHPIZE_DEPS icu-dev
   pecl install ecma_intl-alpha
   ```
2. In a docker image launched with `docker run --rm -it php:8.2-cli-bookworm bash`
   ```
   apt update
   apt install -qy libicu-dev
   pecl install ecma_intl-alpha
   ```

In both cases I have this outout:

```
[... omissis... ]
running: make
/bin/bash /tmp/pear/temp/pear-build-defaultuservJzQtn/ecma_intl-0.1.1/libtool --mode=compile cc -I. -I/tmp/pear/temp/ecma_intl -I/tmp/pear/temp/pear-build-defaultuservJzQtn/ecma_intl-0.1.1/include -I/tmp/pear/temp/pear-build-defaultuservJzQtn/ecma_intl-0.1.1/main -I/tmp/pear/temp/ecma_intl -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/tmp/pear/temp/ecma_intl/src -I/tmp/pear/temp/pear-build-defaultuservJzQtn/ecma_intl-0.1.1/src  -DHAVE_CONFIG_H  -g -O2 -D_GNU_SOURCE    -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/ecma_intl/src/ecma402/calendar.c -o src/ecma402/calendar.lo  -MMD -MF src/ecma402/calendar.dep -MT src/ecma402/calendar.lo
/tmp/pear/temp/pear-build-defaultuservJzQtn/ecma_intl-0.1.1/libtool: line 1290: src/ecma402/calendar.loT: No such file or directory
mkdir src/ecma402/.libs
mkdir: cannot create directory 'src/ecma402/.libs': No such file or directory
make: *** [Makefile:244: src/ecma402/calendar.lo] Error 1
ERROR: `make' failed
```

--- a/config.m4
+++ b/config.m4
@@ -68,6 +68,10 @@ if test "$PHP_ECMA_INTL" != "no"; then
     cxx
   )
 
+  PHP_ADD_BUILD_DIR($ext_builddir/src/ecma402)
+  PHP_ADD_BUILD_DIR($ext_builddir/src/php)
+  PHP_ADD_BUILD_DIR($ext_builddir/src/php/classes)
+
   PHP_REQUIRE_CXX()
   PHP_CXX_COMPILE_STDCXX(11, mandatory, PHP_ECMA_INTL_STDCXX)
 

